gluster-exporter/metric_pv_lv_vg.go

```diff
@@ -0,0 +1,114 @@
+package main
+
```

Comment:
These are peer level metrics so I feel `metric_peer_counts.go` is better name for this file.